spec/models/registration_spec.rb

```diff
@@ -22,4 +22,28 @@
       end
     end
   end
+
+  describe '#destroy_purchased_tickets' do
+    it 'destroys purchased tickets if tickets are purchased' do
+      user = build(:user)
+      conference = build(:conference)
+      registration1 = create(:registration, conference: conference, user: user)
```

Comment:
Same for this line: Please move this line and line 15 into a let statement at the beginning of this file!